build.gradle

```
@@ -38,8 +39,92 @@ tasks.named('bootBuildImage') {
 
 tasks.named('test') {
     useJUnitPlatform()
+    finalizedBy 'jacocoTestReport'
 }
 
+jacoco {
+    toolVersion = "0.8.8"
+}
+
+def jacocoDir = "$buildDir/reports/"
+
+jacocoTestReport {
+    dependsOn test
+    reports {
+        html.required.set(true)
+        xml.required.set(true)
+        csv.required.set(true)
+        html.destination file(jacocoDir + "jacoco/index.html")
+        xml.destination file(jacocoDir + "jacoco/index.xml")
+        csv.destination file(jacocoDir + "jacoco/index.csv")
```

`$buildDir` 제거하면 여기도 변경해주어야합니당
```suggestion
        html.destination jacocoDir.get().file("jacoco/index.html").asFile
        xml.destination jacocoDir.get().file("jacoco/index.xml").asFile
        csv.destination jacocoDir.get().file("jacoco/index.csv").asFile
```